tests/test_filtering.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+from datetime import datetime, timedelta
+from typing import List
+
+from personal_finances.transaction.definition import SimpleTransaction
+from personal_finances.transaction.filtering import transaction_datetime_filter
+
+TEST_TRANSACTIONS_QTD_TO_GENERATE = 300
```

Comment:
I agree, please refer to https://github.com/diegotsutsumi/personal_finances/pull/25#discussion_r1534486004 and check if it could solve this problem on your opinion.